ESLint/.eslintrc

```diff
@@ -1,3 +1,8 @@
 {
```

Comment:
Hmm, ezt a fájlt úgy kellene használni, mint a Lombiq Analyzersben a Build.props-ot: Ezt a fájlt kézzel kelljen mindenkinek egyszer létrehoznia a solutionjében, az `extends`-et beállítva a GE path-ára. Így nem lenne kötelező ugyanolyan mappastruktúrát használni, mint mi. És ezt csak egyszer kell solutionönként (nem projectenként) megcsinálni, és rugalmasságot ad.

Lásd az ehhez kapcsolódó commentemet lejjebb az eslint-base és lombiq-base másolása kapcsán, mert akkor azt se kellene.